bindings/pydeck/pydeck/types/string.py

```diff
@@ -25,3 +25,6 @@ def __eq__(self, other):
 
     def __repr__(self):
         return self.value
+
+    def __hash__(self):
```

Comment:
Where is this used?

Reply:
When checking if a value is in a list of potential values it is using the hash of the object like in this check:
```
    if layer.type_ not in [MapType.QUERY, MapType.TABLE, MapType.TILESET]:
        raise ValueError(f'Invalid layer type selected')
```
If `__hash__` is not implemented, the check will raise an error

Reply:
This requires a new release of pydeck, so I would comment on that code until the new release is available.